apps/next-evans/lib/friend.ts

```diff
@@ -0,0 +1,23 @@
+import type { gender as Gender } from '@prisma/client';
+import { LANG } from '@/lib/env';
+
+export function getFriendUrl(slug: string, gender: Gender): string {
```

Comment:
since i'm having you change the below, can you also indulge me and change this parameter name to `friendSlug` - there are so many kinds of slugs in this domain, i like to err on the side of being explicit/verbose